app/models/commute.rb

```diff
@@ -0,0 +1,7 @@
+class Commute < ActiveRecord::Base
+  validates_presence_of :departed_at
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.